projects/fastify-capture/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@useoptic/fastify-capture",
+  "packageManager": "yarn@3.5.0",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "yarn tsc --build --verbose && yarn babel ./src --out-dir ./build --extensions \".ts\",\".tsx\"",
+    "clean": "rm -rf build",
+    "test": "ENVIRONMENT=test jest --colors --passWithNoTests",
+    "local:run": "yarn ts-node src/index.ts"
+  },
+  "dependencies": {
+    "cookie": "^0.5.0",
+    "fastify": "^4.17.0",
```

Comment:
should this be a dev dep since it's only used for typing? Or should it be a peer dependency?

Reply:
Yeah looks like it's correctly stripped out of the build, let's put it as dev dependency

Reply:
Yeah looks like it's correctly stripped out of the build, let's do this